aptos-move/framework/src/released_framework.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::ReleaseBundle;
+use once_cell::sync::Lazy;
+use std::path::PathBuf;
+
+static TESTNET_RELEASE_BUNDLE: Lazy<ReleaseBundle> = Lazy::new(|| {
+    let path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("releases")
+        .join("testnet.mrb");
```

Comment:
Do we need to update this file manually every time there's a testnet upgrade?

Reply:
Yes, but that is also since our entire release building process is manual. (The head.mrb is manually copied somewhere.) This should be easily scriptable later.